ethereum.star

```diff
@@ -1,45 +1,29 @@
 ethereum_package = import_module(
-    "github.com/kurtosis-tech/ethereum-package/main.star@3.0.0"
+    "github.com/ethpandaops/ethereum-package/main.star@4.3.0"
 )
 
-GETH_IMAGE = "ethereum/client-go:v1.14.0"
-LIGHTHOUSE_IMAGE = "sigp/lighthouse:v5.2.0"
-
 
 def run(plan, args):
     ethereum_package.run(
         plan,
         {
             "participants": [
                 {
-                    # Execution layer (EL)
                     "el_type": "geth",
-                    "el_image": GETH_IMAGE,
-                    # Consensus layer (CL)
                     "cl_type": "lighthouse",
-                    "cl_image": LIGHTHOUSE_IMAGE,
-                    "count": 1,
                 }
             ],
             "network_params": {
-                # The ethereum package requires the network id to be a string.
                 "network_id": str(args["l1_chain_id"]),
                 "preregistered_validator_keys_mnemonic": args[
                     "l1_preallocated_mnemonic"
                 ],
-                "dencun_fork_epoch": 0,
-                "electra_fork_epoch": 100000000,
-                "prague_fork_epoch": 100000000,
```

Comment:
Values already set by default by the package.
https://github.com/ethpandaops/ethereum-package

<img width="810" alt="Screenshot 2024-09-24 at 11 05 17" src="https://github.com/user-attachments/assets/b0c2f621-8734-4bbc-a1c9-64791a77b805">